src/libraries/System.Reflection.Metadata/src/Resources/Strings.resx

```diff
@@ -1,4 +1,64 @@
-﻿<root>
+﻿<?xml version="1.0" encoding="utf-8"?>
+<root>
+  <!-- 
+    Microsoft ResX Schema 
```

Comment:
VS automatically added it and I accidentally let it in; I will remove it later.

Reply:
Reminder to remove this

Reply:
Thanks, I removed it.